### PR TITLE
Added Hindi and Santali language Translations

### PR DESCRIPTION
**Pull request by Prasanta-Hembram**

### CODE REVIEW

**Review by sushain97 — CHANGES_REQUESTED** [review]
@Prasanta-Hembram Thanks for the PR! Please update `locales.json` and then run `make` in `src/strings`. Please update the PR afterwards and I'd be happy to merge.

Apologies for the review delay.

**Comment by Prasanta-Hembram** [comment]
I too was busy for this new year eve no issue :-) . Thanks @sushain97 for suggesting me to make change, i hope that i have done everything good :-). Also, i want to know how to test those translations. Followed this guide https://wiki.apertium.org/wiki/Apertium-html-tools but unable to test my translations. Index file present is the src folder is not working.

**Review by Prasanta-Hembram — COMMENTED** [review]
done

**Comment by sushain97** [comment]
> Also, i want to know how to test those translations. Followed this guide https://wiki.apertium.org/wiki/Apertium-html-tools but unable to test my translations. Index file present is the src folder is not working.

The instructions are here: https://github.com/apertium/apertium-html-tools#contributing

**Comment by Prasanta-Hembram** [comment]
Thanks for the help @sushain97 i encountered some error here, i have successfully installed Node.js and Yarn but after giving  yarn install --dev command i get error which i'm not able to fix. Error is as follows, for which built is not successful. Just want to view the site in html for newly added language, any shortcut ?? or in order to fix what further steps are needed (detailed beginner guide is recommended :-)) 

yarn install --dev
yarn install v1.22.17
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
[2/4] Fetching packages...
[-----------------------------------------------------------------------] 1/757Segmentation fault

**Comment by sushain97** [comment]
> Problem No.1) I have successfully installed Node.js and Yarn but after giving yarn install --dev command i get error which I'm not able to fix. Error is as follows below, for which built is not successful for me. Just want to view the site in html for newly added language, any shortcut ?? or in order to fix what further steps are needed (detailed beginner guide is recommended :-))

Hm, I have a suspicion that you're using Yarn v2 instead of Yarn v1.

You could try these instructions instead: https://github.com/apertium/apertium-html-tools#building.

Combine the `docker` build with `yarn serve` or just directly accessing the built `index.eng.html`.

**Comment by Prasanta-Hembram** [comment]
Thanks @sushain97 for the help i have set Yarn v1 and the build is successfull. I have also updated my translations, and everything is ready now. I hope i have commited well !!

**Comment by Prasanta-Hembram** [comment]
Also, how to translate these languages
Screenshot 2022-01-06 204452
I found https://github.com/apertium/apertium-apy/tree/master/language_names but still confused where to add these language translations. May be most probably in "manual-additions.tsv"?? For Santali language should i create a new tsv file or add it to there too.

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (sushain97) [inline .yarnrc:1]: Please don't add this or the `.yarn` changes :)

Comment (sushain97) [reply .yarnrc:1]: (or `package-json.lock`)

File: src/strings/ara.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
             "ar",
             "ara"
         ],
-        "completion": " 97%  90.10%",
+        "completion": " 97%  90.09%",
```
Comment (sushain97) [inline src/strings/ara.json:11]: Could you back out the trivial changes to these files?

**Review by sushain97 — CHANGES_REQUESTED** [review]
This PR should also not have any changes to `yarn.lock`.

**Comment by sushain97** [comment]
> Also, how to translate these languages Screenshot 2022-01-06 204452 I found https://github.com/apertium/apertium-apy/tree/master/language_names but still confused where to add these language translations. May be most probably in "manual-additions.tsv"?? For Santali language should i create a new tsv file or add it to there too.

Here are the directions: tree/master/language_names#manual---edit

**Comment by Prasanta-Hembram** [comment]
Again, Thanks @sushain97 i will also update those language in apy repo. I will try to correct my error anyhow. I hope i have deleted correct files.

**Comment by Prasanta-Hembram** [comment]
> > Also, how to translate these languages Screenshot 2022-01-06 204452 I found https://github.com/apertium/apertium-apy/tree/master/language_names but still confused where to add these language translations. May be most probably in "manual-additions.tsv"?? For Santali language should i create a new tsv file or add it to there too.
> 
> Here are the directions: https://github.com/apertium/apertium-apy/tree/master/language_names#manual---edit

Manual - EDIT

    **manual-fixes.tsv** contains fixes to the scraped data.
    **manual-additions.tsv** contains additions to the scraped data.
    variants.tsv contains fixes/additions to the scraped data for language variants.
    turkic.tsv contains Turkic language names.

N.B. Due to historical reasons, there are currently some additions in manual-fixes.tsv.

So, HIndi  language locale translation should go to **manual-fixes.tsv** and Santali language locale translations (new language in apy) should be in **manual-additions.tsv** or both should be in which files if I'm wrong??

**Comment by sushain97** [comment]
> So, HIndi language locale translation should go to **manual-fixes.tsv** and Santali language locale translations (new language in apy) should be in **manual-additions.tsv** or both should be in which files if I'm wrong??

Updates to existing language names should go in `fixes` and new one should go in `additions`.

File: .yarnrc (hunk 1 above)
Comment (sushain97) [reply .yarnrc:1]: This file should be deleted as well as the changes to `yarn.lock` and `package.json`.

**Comment by Prasanta-Hembram** [comment]
HI, @sushain97 can you check again i have made some changes.

**Comment by sushain97** [comment]
Could you revert the changes to `package.json` and `yarn.lock`?

**Comment by Prasanta-Hembram** [comment]
> `package.json` and `yarn.lock`?

can you again check?

**Review by sushain97 — APPROVED** [review]

**Pull request merged**